app/scripts/modules/google/src/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -379,14 +383,14 @@ module.exports = angular
 
       function buildServerGroupCommandFromExisting(application, serverGroup, mode) {
         mode = mode || 'clone';
-        const serverGroupName = NameUtils.parseServerGroupName(serverGroup.name);
+        const moniker = serverGroup.moniker;
```

Comment:
What is the fallback behavior here if the previous server group was deployed before moniker support? Is this all handled in the Clouddriver changes?

Reply:
Yes, all server groups will have moniker set by Clouddriver.